src/shared/styles/index.js

```diff
@@ -0,0 +1,15 @@
+import colors from './colors';
+import fonts from './fonts';
+import layout from './layout';
+
+export {
+    colors,
+    fonts,
+    layout,
+};
+
+export default {
```

Comment:
I'd probably not use a default export here since we already have named exports above.

Reply:
I'm keeping the default export as to import in the theme provider you only need to ```import styles from '..styles'` instead of specifying every thing you want to export and put them into an object. Also, if you want to add something else to the styles you will need to add it in the theme provider whereas with the default export you only need to set it in the default export

Reply:
import * as styles from '...' is pretty much the same. This is my own opinion, but I tend to prefer to follow a single approach to export functions or components. It causes much less confusion. If we're only using the default one, I don't see any reason to keep the named exports around then. We will be not cherry picking styles anyway, right?

Reply:
Removed the default export.